doc/api/repl.md

```diff
@@ -178,34 +174,6 @@ function myEval(cmd, context, filename, callback) {
 repl.start({prompt: '> ', eval: myEval});
 ```
 
-#### Recoverable Errors
```

Comment:
I for one am not very happy this feature got removed. In recent versions of Chrome, a similar function just got added (instead of having to type <kbd>Shift</kbd>+<kbd>Enter</kbd>, the prompt automatically recognizes line continuation), and to have to type an awkward <kbd>Ctrl</kbd>+<kbd>J</kbd> for the same feature does not sound very appealing to me.

If this change is here to stay, may I suggest we at least add support to <kbd>Shift</kbd>+<kbd>Enter</kbd> also/instead? It is the syntax in most IM apps, and also in most developer consoles.